taglib/mp4/mp4tag.cpp

```diff
@@ -74,9 +74,20 @@ MP4::Tag::Tag(TagLib::File *file, MP4::Atoms *atoms) :
             atom->name == "hdvd" || atom->name == "shwm") {
       parseBool(atom);
     }
-    else if(atom->name == "tmpo" || atom->name == "rate" || atom->name == "\251mvi" || atom->name == "\251mvc") {
+    else if(atom->name == "tmpo" || atom->name == "\251mvi" || atom->name == "\251mvc") {
       parseInt(atom);
     }
+    else if(atom->name == "rate") {
+      AtomDataList data = parseData2(atom);
```

Comment:
I believe this should be `parseData` instead of `parseData2`

Reply:
Obviously not - parseData returns a ByteVectorList, which loses the Atom type which is required here.